ui/src/main.rs

```diff
@@ -42,7 +42,7 @@ use crate::layouts::unlock::UnlockLayout;
 use crate::state::ui::WindowMeta;
 use crate::state::Action;
 use crate::state::{friends, storage};
-use crate::utils::sounds::Sounds;
+//use crate::utils::sounds::Sounds;
```

Comment:
Let's just remove this line for now, We can always add it back in if we need it in the future. Especially for imports.

Reply:
totally get it! fixed!